package.json

```diff
@@ -35,16 +35,16 @@
     "react-transition-group": "^2.4.0",
     "react-window-decorators": "^1.0.2",
     "styled-animate": "^0.2.0",
-    "styled-components": "^3.4.2",
+    "styled-components": "^4.4.0",
```

Comment:
I updated the deps because it was causing an error. It had something to do with the deprecated `injectGlobal` from styled-components.